.gitignore

```diff
@@ -1593,3 +1593,66 @@ node_modules/xdg-basedir/index.js
 node_modules/xdg-basedir/license
 node_modules/xdg-basedir/package.json
 node_modules/xdg-basedir/readme.md
+node_modules/axios/CHANGELOG.md
```

Comment:
This shouldn't be here at all.
You can just write:
node_modules/

And that's it, it will consider inner files as well.